### PR TITLE
[NET] Non-running dns servers removed from chainParams.

### PR DESCRIPTION
**Pull request by furszy**
Removed non-running DNS for mainnet and testnet.

### CODE REVIEW

**Comment by Fuzzbawls** [comment]
Could also remove the static IP addresses, and I believe both @Warrows and @random-zebra have proper DNS seeders running that could be added

**Comment by furszy** [comment]
k, just texted coin-server about his server too. If there is no answer then that can be removed too.

File: src/chainparams.cpp [hunk 1]
```diff
@@ -199,7 +199,6 @@ class CMainParams : public CChainParams
         vSeeds.push_back(CDNSSeedData("fuzzbawls.pw", "pivx.seed.fuzzbawls.pw"));     // Primary DNS Seeder from Fuzzbawls
         vSeeds.push_back(CDNSSeedData("fuzzbawls.pw", "pivx.seed2.fuzzbawls.pw"));    // Secondary DNS Seeder from Fuzzbawls
         vSeeds.push_back(CDNSSeedData("coin-server.com", "coin-server.com"));         // Single node address
```
Comment (Fuzzbawls) [inline src/chainparams.cpp]: Remove this entry as well

**Comment by Warrows** [comment]
You can add `testnet.dnsseed.pivx.warrows.dev` for testnet and `dnsseed.pivx.warrows.dev` for mainnet.

**Comment by furszy** [comment]
Changes completed.

**Pull request merged**